samples/RaspberryPiInstructions.md

```diff
@@ -26,6 +27,8 @@ These steps have been tested on a RPi 2 and RPi 3 with Linux and Windows.
 </Project>
 ```
 
+* Add a new package source to the project with source: `https://dotnet.myget.org/F/dotnet-core/api/v3/index.json`.
```

Comment:
Today I tried the steps over again.

So I reinstalled Microsoft .NET Core 2.0.0 - SDK Preview 1. I expected `2.0.0` when running the `dotnet --version` command,  however it returned `1.0.0`. I used the following link to install the .NET Core SDK: [Windows x64 - Installer](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/master/dotnet-dev-win-x64.latest.exe)

> Probably using the 1.0 SDK?

So yes, you are correct. I was not presented with a NuGet.config file probably due to the fact that I was using .NET Core 1 SDK. Others could be running into the same issue. I had to manually add the package source with Visual Studio 2017. Perhaps the first line should be elaborated on, since it only points to the repository without giving a 'how-to':
>Install .NET Core 2.0 SDK into a supported developer configuration

Creating multiple paths for multiple environments could be an option, but could confuse the reader even more. I'm not sure what the correct approach is.